internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixReadOnlySchemaRepository.java

```diff
@@ -163,7 +163,10 @@ SchemaEntry forceRefreshSupersetSchemaWithRetry(String storeName) {
     Store store = getStoreRepository().getStore(storeName);
     int supersetSchemaId = store.getLatestSuperSetValueSchemaId();
     AtomicReference<SchemaEntry> supersetSchemaEntry = new AtomicReference<>();
-    RetryUtils.executeWithMaxAttempt(() -> {
+    Long currentTimestamp = System.currentTimeMillis();
```

Comment:
nit: Use long instead of Long to avoid auto boxing

Reply:
Fixed